src/handlers/flow/FlowStreamHandler.cpp

```diff
@@ -14,15 +14,125 @@
 
 namespace visor::handler::flow {
 
-template <typename Out>
-static void split(const std::string &s, char delim, Out result)
+static std::pair<bool, std::vector<Ipv4Subnet>::const_iterator> match_subnet(std::vector<Ipv4Subnet> &ipv4_list, uint32_t ipv4_val)
```

Comment:
we have these matching and parsing functions in many places too by now, don't we? can we move to a common utility library for it?

Reply:
Yes, I think we have this in 3 placas... só it would be Better to be in a lib

Reply:
I did it